BT-Tracking/Interface/Current Data/CurrentData.swift

```diff
@@ -19,6 +20,14 @@ final class CurrentDataRealm: Object {
     @objc dynamic var deceasedTotal: Int = 0
     @objc dynamic var currentlyHospitalizedTotal: Int = 0
 
+    @objc dynamic var appDate: Date?
+    @objc dynamic var activationsYesterday: Int = 0
+    @objc dynamic var activationsTotal: Int = 0
+    @objc dynamic var keyPublishersYesterday: Int = 0
+    @objc dynamic var keyPublishersTotal: Int = 0
+    @objc dynamic var notificationsYesterday: Int = 0
+    @objc dynamic var notificationsTotal: Int = 0
```

Comment:
Nechal bych to ve vlastnim Realm modelu. Neni to stejna vec, i kdyz je to na stejne strance, a navic ma kazda vlastni endpoint.

Reply:
Me prave pak prislo ze by to pridalo jen dalsi kod na ukladani realm objektu kvuly cache

Reply:
To by pridalo, ale to mi prijde v poradku. IMO by to melo byt samostatne.

Reply:
Udelal jsem tam takovy kompromis, prijde ti to lepsi?

Reply:
Je to lepsi, ale porad se mi zda zbytecne michat 2 veci do jednoho Realm objektu.